solidity/contracts/INetworkSettings.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: SEE LICENSE IN LICENSE
 pragma solidity 0.6.12;
 
-import "./utility/interfaces/ITokenHolder.sol";
+import "./token/interfaces/ITokenHolder.sol";
```

Comment:
ITokenHolder isn't a token, why do you think it should be under the token folder?

Reply:
I wanted to group every "token utilities" together. Do you think it should go back under `utility`?

Reply:
yeah, I think we should only keep tokens under the token folder, many other contracts might be considered "token utilities" - the token holder is just a wallet.

Reply:
Moved back 😎